tests/prague/eip7692_eof_v1/tracker.md

```diff
@@ -0,0 +1,25 @@
+# EOF Testing Coverage Tracker
+
+## EIP-3540: EOF - EVM Object Format v1
+
+- [ ] Example Test Case 1
```

Comment:
Do we want multiple statuses?

- todo
- in ethereum/tests
- done

Reply:
Should we also write where the "done" test is?

Reply:
Added!